app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/PlayerAlbumCoverFragment.java

```diff
@@ -163,6 +181,22 @@ public void run() {
                 .start();
     }
 
+    public void setSynchronizedLyrics(AbsSynchronizedLyrics sLyrics)
```

Comment:
This makes me think you're a C developer. 😛 

It'd be nice if the style matched the rest of the code base (i.e. curly braces on the same line).